internal/models/users/users.go

```diff
@@ -191,3 +191,12 @@ func AssignOptInRankingToUser(db *sql.DB, userID uuid.UUID, optInRanking bool) e
 	)
 	return err
 }
+
+func DeleteUserByUUID(db *sql.DB, userID uuid.UUID) error {
```

Comment:
By ID; UUID is the data type and should be irrelevant for the function name?

Reply:
UUID is also the actual name of the id we are using, so it doesnt really matter which one we use, as long as it's consistent all over the codebase

Reply:
Everywhere else we use ID

Reply:
done